packages/webview_flutter/webview_flutter_wkwebview/darwin/Classes/FWFInstanceManager.m

```diff
@@ -140,17 +140,17 @@ - (long)identifierWithStrongReferenceForInstance:(nonnull NSObject *)instance {
   NSNumber *__block identifierNumber = nil;
   dispatch_sync(_lockQueue, ^{
     identifierNumber = [self.identifiers objectForKey:instance];
-    if (identifierNumber) {
+    if (identifierNumber != nil) {
       [self.strongInstances setObject:instance forKey:identifierNumber];
     }
   });
-  return identifierNumber ? identifierNumber.longValue : NSNotFound;
+  return identifierNumber != nil ? identifierNumber.longValue : NSNotFound;
 }
 
 - (BOOL)containsInstance:(nonnull NSObject *)instance {
   BOOL __block containsInstance;
   dispatch_sync(_lockQueue, ^{
-    containsInstance = [self.identifiers objectForKey:instance];
+    containsInstance = [self.identifiers objectForKey:instance] != nil;
```

Comment:
nit: (probably unrelated to this pr so no need to change): can just use pthread or NSLock if `__block` feels tedious to write. 

Using queues as mutex locks feels a bit overkill although many people are doing it so 🤷. iirc i ran a benchmark super long time ago and locks (even the slow NSLock) are A LOT faster than queues.